Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM amazoncorretto:17-alpine3.19-jdk
```

Comment:
I change this image because [openJDK](https://hub.docker.com/_/openjdk) is deprecated.